Microsoft.Toolkit.Uwp.UI/Extensions/ConditionalToolTip.cs

```diff
@@ -0,0 +1,76 @@
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.UI.Extensions
+{
+    /// <summary>
+    /// Provides attached dependency properties that allow a <see cref="UIElement"/>
+    /// to conditionally enable or disable its tooltip.
+    /// </summary>
+    public class ConditionalToolTip : DependencyObject
```

Comment:
This can just be a partial class off our existing `UIElementExtensions` class, it doesn't need to be its own `DependencyObject`.